lib/common/utils.js

```diff
@@ -140,9 +149,9 @@ const normalize = string => string.replace(reqSep, sysSep)
 
 export const r = function r() {
```

Comment:
Can we use `export const r = (...args) => {` yet? How is support?

Reply:
We can't as node 8.0.0 doesn't support spread/rest object (https://node.green/#ES2018-features-object-rest-spread-properties)

Reply:
That's not an object though, that's an array. So, much earlier support.

Reply:
Whoops. You are right (https://node.green/#ES2015-syntax-rest-parameters-arguments-object-interaction)

Reply:
That would be nice, since further on even the actual array is being mutated.